test/integration/developer_can_login_to_account_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.